requirements.txt

```diff
@@ -1 +1,23 @@
+apt-xapian-index==0.44
+chardet==2.0.1
+click==6.2
+cligj==0.3.0
+command-not-found==0.2.44
+Fiona==1.5.1
+GDAL==1.10.0
+GnuPGInterface==0.3.2
+language-selector==0.1
 nose==1.3.4
+numpy==1.10.1
+pandas==0.17.1
+pycurl==7.19.0
+pyshp==1.2.3
+python-apt===0.8.3ubuntu7
+python-dateutil==2.4.2
+python-debian===0.1.21ubuntu1
+pytz==2015.7
+scipy==0.16.1
+Shapely==1.2.19
+six==1.10.0
+ufw==0.31.1.post1
+unattended-upgrades==0.1
```

Comment:
According to the author, only
```
sudo pip install fiona 
sudo pip install numpy
sudo pip install pandas
sudo pip install shapely
sudo apt-get install libblas-dev liblapack-dev libatlas-base-dev gfortran
sudo pip install scipy
sudo pip install pyshp
```
should be required for the python dependencies.